test/utils/test-utils.js

```diff
@@ -47,6 +47,23 @@ const processKill = (process) => {
     }
 };
 
+const getUniqueDirectory = () => path.resolve(os.tmpdir(), Math.random().toString(36).substr(2, 9));
+
+const copyRecursiveSync = function (src, dest) {
+    if (fs.existsSync(src) && fs.statSync(src).isDirectory()) {
+        fs.mkdirSync(dest);
+        fs.readdirSync(src).forEach((childItemName) => {
```

Comment:
```suggestion
        fs.readdirSync(src).forEach((childDirectory) => {
```

Reply:
It is not only file, here and dir and link

Reply:
childINode then